components/storage/blockdevice/COMPONENT_SD/config/mbed_lib.json

```diff
@@ -60,12 +60,6 @@
              "SPI_CLK":  "PTD5",
              "SPI_CS":   "PTD4"
         },
```

Comment:
Please remove all NUCLEO and DISCO target_overrides configuration
as SPI_xxx define should be already defined for them.
Thx

Reply:
ASFAIK, not all NUCLEO and DISCO has those pins defined. Also, some of them were defined by the data from spif and some from sd.  So in case, spif is the original definition we left the sd target override as it was.